tools/pot/openvino/tools/pot/graph/special_patterns.py

```diff
@@ -256,3 +256,52 @@ def create_equal_logicalnot_pattern():
     pattern.append_single_op('Equal', 'equal')
     pattern.append_single_op('LogicalNot', 'logicalnot')
     return pattern.set_name('equal_logicalnot').pattern
+
+
+@registry_ignore_patterns('inputs')
```

Comment:
@IndiraSalyahova, @KodiaqQ, any chance we can parametrize the presence of Concat in the existing input-related patterns? or do we need a separate set of patterns as it is done in this PR?
If the latter is right, I would recommend moving and placing all these patterns right after the first set of input-related patterns. The same is applicable for documentation.

Reply:
In the current implementation, we will need to create a separate set of input patterns with the Concat. We can iterate over different types of layers when creating a pattern, but on the condition that the layer is exactly in the pattern. Here is the case when the layer may or may not be.

Reply:
Do we need to create the model examples with the corresponding patterns for our graph tests? Or are they already presented, @IndiraSalyahova?

Reply:
Let's validate at least the basic ones where we have intput->transpose->ScaleShift

Reply:
@IndiraSalyahova, can you please introduce the test and fix CI we need it asap.